packages/browser-lite/src/wasm_exec.ts

```diff
@@ -12,6 +11,7 @@
  * NOTE(jalextowle): This comment must be here so that typedoc knows that the above
  * comment is a module comment
  */
+/* tslint:disable */
```

Comment:
Maybe I'm missing something but I don't think this change is necessary.

Based on https://palantir.github.io/tslint/usage/rule-flags/:

> /* tslint:disable */ - Disable all rules for the rest of the file


We don't need to move the `tslint:disable` comment nor do we need to add `tslint:enable` at the end of the file.

Reply:
I was just moving it because the `NOTE` is now in a better location. I removed the `tslint:enable` though